packages/core/src/postprocessors/rerankers/JinaAIReranker.ts

```diff
@@ -0,0 +1,89 @@
+import { getEnv } from "@llamaindex/env";
+import type { NodeWithScore } from "../../Node.js";
+import { MetadataMode } from "../../Node.js";
+import type { BaseNodePostprocessor } from "../types.js";
+
+interface JinaAIRerankerResult {
+  index: number;
+  document?: {
+    text?: string;
+  };
+  relevance_score: number;
+}
+
+export class JinaAIReranker implements BaseNodePostprocessor {
+  model: string = "jina-reranker-v1-base-en";
+  topN?: number;
+  apiKey?: string = undefined;
+
+  constructor(init?: Partial<JinaAIReranker>) {
+    this.topN = init?.topN;
```

Comment:
```suggestion
    this.topN = init?.topN ?? 2;
```

Should we add default value for the topN parameter, like python version?

https://github.com/run-llama/llama_index/blob/aa13d47444692faa06b5753b7451b1920837b29c/llama-index-integrations/postprocessor/llama-index-postprocessor-jinaai-rerank/llama_index/postprocessor/jinaai_rerank/base.py#L26

Reply:
Thanks for the comment! In fact we do not require topN in the api, the default value will be len(docs) if it's not provided. I will discuss whether we want a good value for it with my colleagues.